src/app/components/applications/SSV/MyAccount/components/EditOperatorDetails/InputFieldComponent.tsx

```diff
@@ -14,8 +16,25 @@ const InputFieldComponent = ({ fieldKey, extendClass, placeholder  }: { fieldKey
 
     const onChangeHandler = (event: any) => {
         const { value } = event.target;
-        setCurrentValue(value.length > 1 ? value : value.trim());
-        metadataStore.setMetadataValue(fieldKey, value.trim());
+
+        if (fieldKey === FIELD_KEYS.DKG_ADDRESS) {
+            // Ensure http:// is always present
+            if (!value.startsWith(HTTP_PREFIX)) {
+                setCurrentValue(HTTP_PREFIX);
```

Comment:
Looking at PRD I think it should be:
1. Input with addon (look here: https://getbootstrap.com/docs/4.0/components/input-group/) 
2. Because addon is always there - placeholder should be everything except schema
3. Addon should be with transparent background to make feel like it's part of the text
4. When user types the rest in input field, or insert it, his input should be cleaned up from schema
5. But when final value is collected to use for signature and to send to backend - schema should be added before